apps/server/src/modules/medium-metadata/strategy/bilo.strategy.spec.ts

```diff
@@ -76,18 +77,17 @@ describe(BiloStrategy.name, () => {
 			const setup = () => {
 				const mediumId = 'medium-id';
 				const mediaSource = mediaSourceFactory.withBildungslogin().build();
-
-				biloMediaClientAdapter.fetchMediaMetadata.mockResolvedValueOnce([]);
+				biloMediaClientAdapter.fetchMediumMetadata.mockRejectedValueOnce(new BiloNotFoundResponseLoggableException());
 
 				return { mediumId, mediaSource };
 			};
 
 			it('should throw not found exception', async () => {
 				const { mediumId, mediaSource } = setup();
 
-				const result = strategy.getMediumMetadataItem(mediumId, mediaSource);
+				const promise = strategy.getMediumMetadataItem(mediumId, mediaSource);
 
-				await expect(result).rejects.toThrow(MediumMetadataNotFoundLoggableException);
+				await expect(promise).rejects.toThrow(new MediumMetadataNotFoundLoggableException(mediumId, mediaSource.id));
```

Comment:
this test is not working correctly. The promise throws a BiloNotFoundResponseLoggableException but passes when you compare it to a MediumMetadataNotFoundLoggableException. Try using `.rejects.toThrow(MediumMetadataNotFoundLoggableException)` as syntax again for it to work correctly.